src/AudioIO.cpp

```diff
@@ -1748,12 +1774,28 @@ AudioThread::ExitCode AudioThread::Entry()
          gAudioIO->TrackBufferExchange();
          gAudioIO->mAudioThreadShouldCallTrackBufferExchangeOnce
             .store(false, std::memory_order_relaxed);
+
+         lastCommand = Command::eOnce;
       }
       else if( gAudioIO->mAudioThreadTrackBufferExchangeLoopRunning
          .load(std::memory_order_relaxed))
       {
          gAudioIO->TrackBufferExchange();
+
+         lastCommand = Command::eLoopRunning;
       }
+      else
+      {
+         if (lastCommand == Command::eLoopRunning)
+         {
+            // Main thread has told us to stop; (actually: to neither process "once" nor "loop running")
+            // acknoweldge that we received the order, and no more processing will be done.
```

Comment:
Spelling, "acknowledge"

Reply:
Fine by me. I am going to write many more things, so I will organize them in multiple commits for better clarity.

Reply:
ok, done